src/modules/data/audioStream.c

```diff
@@ -1,9 +1,11 @@
 #include "data/audioStream.h"
 #include "data/blob.h"
+#include "data/soundData.h"
 #include "core/ref.h"
 #include "core/util.h"
 #include "lib/stb/stb_vorbis.h"
 #include <stdlib.h>
+#include <memory.h>
```

Comment:
Whoa, I had never heard of `memory.h`.  Is it any different from `string.h`?  We're using that for memcpy etc. everywhere else.

Reply:
uhhh I don't know what vestige of my brain that I dug this out from; I just wanted memcpy! It doesn't seem to be posix or anything?? I'l change to string.h